core/templates/directives/focus-on.directive.ts

```diff
@@ -16,10 +16,57 @@
  * @fileoverview FocusOn Directive (not associated with reusable
  * components.)
  * NB: Reusable component directives should go in the components/ folder.
+ *
+ * NOTE: PLEASE MAKE SURE THAT ANY CHANGES MADE TO THE CODE IS DONE TO BOTH THE
```

Comment:
NOTE --> NOTE TO DEVELOPERS

You can put everything else in regular case. Maybe capitalize just "BOTH", or put asterisks before and after it.

"changes made to the code is done to" --> "code changes are done for"

Reply:
Done!